src/plays/personal-profile-card/components/profile-form.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react'
+
+import ProfileType from '../types'
+
+interface Props {
+  value: ProfileType
+  profile: ProfileType
+  onChange: React.ChangeEventHandler
+  onClick: () => void
+  onUpload: React.ChangeEventHandler
+  onClear: () => void
+}
+
+const ProfileForm = ({
+  value,
+  profile,
+  onChange,
+  onClick,
+  onUpload,
+  onClear,
+}: Props) => {
+  return (
+    <div className="profile-form md:mr-5">
+      <div className="profile-info">
+        <input
+          className="cover-input mx-auto mr-0 relative top-[20px] mt-[-20px] h-[30px] sm:h-[40px] w-[30px] sm:w-[40px] rounded-full flex bg-[#45576f] hover:cursor-pointer"
+          type="file"
+          name="cover"
+          onChange={onUpload}
+        />
+        <img
+          src={
+            !value || !value.cover
```

Comment:
can't we check here `value?.cover` with this expression? if not what error/issue and if yes please replace all other checks with this expression

Reply:
Changed to `value.cover === ''`. Because default state define with empty string. The other checks also changed.